### PR TITLE
Feautre/setup task

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: lib/tasks/setup.rake [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+# Everything that needs to be done before you can use this app
```
Comment (Ana06) [inline lib/tasks/setup.rake]: Why is this needed? In my opinion the `setup` name implies this already

File: lib/tasks/setup.rake [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    copy_example_file(".env.#{Rails.env}", 'dotenv.example')
+  end
+
+  desc 'Bootstrap the application'
+  task bootstrap: [:configure, :environment] do
+    puts 'Creating the database...'
+    begin
+      # Only do this if the database does not exist...
+      Rake::Task['db:version'].invoke
+      # rubocop:disable Style/RescueStandardError
```
Comment (Ana06) [inline lib/tasks/setup.rake]: Why is that needed? Using `rescue` should be the default in Rubocop :confused:

Comment (hennevogel) [reply lib/tasks/setup.rake]: This is disabling the reporting the `rescue` in the middle :)

Comment (Ana06) [reply lib/tasks/setup.rake]: @hennevogel that is what it looks like, but I think it shouldn't be needed, because the cop is enforcing `rescue` and not complaining about it (at least is what the documentation says). :confused:

Comment (hennevogel) [reply lib/tasks/setup.rake]: No it enforces rescuing `StandardError` instead of all exceptions (like we do in this case)
